src/Microsoft.Health.Dicom.Core/Features/Common/DicomFileNameWithPrefix.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Features.Model;
+
+namespace Microsoft.Health.Dicom.Core.Features.Common;
+public class DicomFileNameWithPrefix : IDicomFileNameBuilder
```

Comment:
This seems leaky to have it out in the core. It doesn't seem like the core library should concern itself with the naming convention used in the blob store.

Reply:
Moved to blob